core/java/iesi-rest-without-microservices/src/test/java/io/metadew/iesi/server/rest/script/ScriptControllerTest.java

```diff
@@ -313,6 +310,175 @@ void getAllPaginationTest() throws Exception {
 
     }
 
```

Comment:
Can you also create a test that contains all entries on 1page

Reply:
These are Mocked data and thus will appear as I created them.
I will do it there and in the service to test the interaction with DB.